### DISCUSSION TITLE
mix background image and gradient

### DISCUSSION THREAD

**Discussion by kertahayu** [tailwindlabs/tailwindcss#2707] [Help]
I saw this documentation
```
module.exports = {
    theme: {
      extend: {
        backgroundImage: theme => ({
           'hero-pattern': "url('/img/hero-pattern.svg')",
           'footer-texture': "url('/img/footer-texture.png')",
        })
      }
    }
  }
```
how do i mix this with a gradient?

so it might look something like this
```
<div class="bg-hero-pattern bg-gradient-to-l from-red-500 to-blue-500 bg-blend-color-burn ">
```

bg-blend-color-burn from hacknug/tailwindcss-blend-mode

**Comment by kertahayu** [comment]
> I saw this documentation
> 
> ```
> module.exports = {
>     theme: {
>       extend: {
>         backgroundImage: theme => ({
>            'hero-pattern': "url('/img/hero-pattern.svg')",
>            'footer-texture': "url('/img/footer-texture.png')",
>         })
>       }
>     }
>   }
> ```
> 
> how do i mix this with a gradient?
> 
> so it might look something like this
> 
> ```
> <div class="bg-hero-pattern bg-gradient-to-l from-red-500 to-blue-500 bg-blend-color-burn ">
> ```
> 
> bg-blend-color-burn from hacknug/tailwindcss-blend-mode


For a while I added a linear-gradient(to-left, var(--gradient-color-stops)), because this is what I need. I hope there are other inputs to make what I mean. like changing it from left to right.

```
backgroundImage: theme => ({
       'hero-pattern': "url('/img/hero-pattern.svg'), linear-gradient(to left, var(--gradient-color-stops))",
       'footer-texture': "url('/img/footer-texture.png')",
 })
``

**Reply by laurent512** [reply]
Is there any better solution today than this ? Thank you so much   ! :)
